plugin/path_login.go

```diff
@@ -428,6 +437,25 @@ func (b *GcpAuthBackend) pathGceLogin(req *logical.Request, loginInfo *gcpLoginI
 	return resp, nil
 }
 
+func authMetadata(loginInfo *gcpLoginInfo, serviceAccount *iam.ServiceAccount) map[string]string {
```

Comment:
Not that I see a problem currently, but keep in mind that all the items here will be audit logged in plaintext.

Reply:
👍